celery/local.py

```diff
@@ -395,14 +384,11 @@ def maybe_evaluate(obj):
 
 DEFAULT_ATTRS = {'__file__', '__path__', '__doc__', '__all__'}
 
+
 # im_func is no longer available in Py3.
 # instead the unbound method itself can be used.
-if sys.version_info[0] == 3:  # pragma: no cover
-    def fun_of_method(method):
-        return method
-else:
-    def fun_of_method(method):  # noqa
-        return method.im_func
+def fun_of_method(method):
```

Comment:
I think this is only used [here](https://github.com/celery/celery/blob/6215f34d2675441ef2177bd850bf5f4b442e944c/celery/local.py#L498), so perhaps we can skip the function all together and just inline it?